week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/shop/cart-item.component.ts

```diff
@@ -0,0 +1,25 @@
+import { div } from "../framework/dom-creators";
+
+export type CartItem = {
+  name: string;
+  amount: number;
+  unit: string;
+  price: {
```

Comment:
Jakbyś tutaj @wszczepanski97 miał model `Price` to można by było go też wykorzystać do: [cart-item-sum.component.ts](https://github.com/michaljabi/js-na-frontach-zadania/pull/128/files#diff-f383d63a5483783b72d2d0aca6feb97c6281d1e6a1e48dd1119247fb02c99104)